src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -79,7 +100,8 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
 
     private _setupPayment(
         address?: Address | null,
-        onError?: (error: BraintreeError | StandardError) => void
+        onError?: (error: BraintreeError | StandardError) => void,
+        paypalCheckoutInstance?: any
```

Comment:
Can you provide a type for `paypalCheckoutInstance` instead of using `any`?

Reply:
fixed